sdks/python/apache_beam/io/gcp/bigquery.py

```diff
@@ -1644,6 +1650,7 @@ def __init__(
     else:
       self.schema = bigquery_tools.get_dict_table_schema(schema)
     self.batch_size = batch_size
+    self.batch_duration = batch_duration
```

Comment:
Does it make sense to add a check to make sure usewrs are only providing this argument when it's usable? (and throw ValueError otherwise?)

Reply:
Sure, I'll add it